packages/mutt.rb

```diff
@@ -1,50 +1,46 @@
-  version '2.2.10'
+  version '2.2.11'
   license 'GPL-2'
   compatibility 'all'
-  source_url 'ftp://ftp.mutt.org/pub/mutt/mutt-2.2.10.tar.gz'
-  source_sha256 '4d773f22422f79096f7b94b57bee45654ad9a25165dbb36463c58295b4cd3d88'
+  source_url 'ftp://ftp.mutt.org/pub/mutt/mutt-2.2.11.tar.gz'
+  source_sha256 '12325cf66d5ff8ac4bd87fac8db52c869de52dd278fc301cfd57d5a1f9f465cc'
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mutt/2.2.10_armv7l/mutt-2.2.10-chromeos-armv7l.tar.zst',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mutt/2.2.10_armv7l/mutt-2.2.10-chromeos-armv7l.tar.zst',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mutt/2.2.10_i686/mutt-2.2.10-chromeos-i686.tar.zst',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mutt/2.2.10_x86_64/mutt-2.2.10-chromeos-x86_64.tar.zst'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mutt/2.2.11_armv7l/mutt-2.2.11-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mutt/2.2.11_armv7l/mutt-2.2.11-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mutt/2.2.11_i686/mutt-2.2.11-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mutt/2.2.11_x86_64/mutt-2.2.11-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: '1d113f20e2ca2f4adde9a207e60978a20b1172b18b901be7837dd6122dcdaea0',
-     armv7l: '1d113f20e2ca2f4adde9a207e60978a20b1172b18b901be7837dd6122dcdaea0',
-       i686: '3a831a695aa4a276863cf43ca9f72c47942da7d299e0ae4c82c6310edddd8553',
-     x86_64: '0e325a6a0e00e42b8b339b599c57641a28724fa2d3574b26112f8be9e2a4b785'
+    aarch64: '98bc06c612dcc678fc32265f832e749660413a316b1f8c319ade106cef04011e',
+     armv7l: '98bc06c612dcc678fc32265f832e749660413a316b1f8c319ade106cef04011e',
+       i686: 'ae547bc244cbd265563341f9a93dc221ffc552ec6538544167be22b2e842f105',
+     x86_64: '31e4c665796270c5a24bc6697ce01e71f5fd8f355ad4fb66eb27c200573b9377'
   })
 
   depends_on 'libcyrussasl'
   depends_on 'libxslt'
 
-  def self.build
-    system "./configure #{CREW_OPTIONS} \
-            --with-mailpath=#{CREW_PREFIX}/etc/mail \
-            --with-sasl=#{CREW_LIB_PREFIX}/sasl2 \
-            --with-ssl \
-            --enable-imap \
-            --enable-smtp \
-            --enable-hcache"
-    system 'make'
+  configure_options "--with-mailpath=#{CREW_PREFIX}/etc/mail \
+    --with-sasl=#{CREW_LIB_PREFIX}/sasl2 \
+    --with-ssl \
+    --enable-imap \
+    --enable-smtp \
+    --enable-hcache"
+
+  run_tests
+
+  def self.install
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/mail"
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d/"
-    @muttenv = <<~MUTTEOF
+    File.write "#{CREW_DEST_PREFIX}/etc/env.d/mutt", <<~MUTTEOF
```

Comment:
Should we use FileUtils.install for this file?

Reply:
I suppose but how would that work with a heredoc?  I figured since we don't need execute permissions on this file, the default read/write perms would be acceptable.

Reply:
Perms should be 0644, right?

Reply:
Yes, I believe so.